api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirEncounterDaoImpl.java

```diff
@@ -101,4 +104,10 @@ protected String paramToProp(@Nonnull String param) {
 				return null;
 		}
 	}
+	
+	@Override
+	protected Criterion generateNotCompletedOrderQuery(String path) {
+		// not implemented in Core until 2.2; see override in FhirEncounterDaoImpl_2_2
+		return null;
```

Comment:
@mseaton  @ibacher just return null here... lmk if we should log something or take some other action